rca/project_styleguide/templates/patterns/molecules/key-details/key-details--event.html

```diff
@@ -116,5 +116,18 @@ <h3 class="body body--two key-details__sub-heading">Read time</h3>
             </ul>
         </div>
     {% endif %}
-
+    
+    {% if page.asset_heading and page.asset_link_text and page.asset_link %}
+         <div class="key-details__section key-details__section--press-pack">
+            <h3 class="body body--two key-details__sub-heading">{{ page.asset_heading }}</h3>
+            <ul class="key-details__list">
+                <li class="key-details__list-item">
+                    <a href="{{ page.asset_link }}" class="link link--tertiary link--link link--external" target="_blank">
+                        <span class="link__label">{{ page.asset_link_text }}</span>
```

Comment:
```suggestion
                        <span class="link__label">{% firstof page.asset_link_text 'Download' %}</span>
```
Just because none of these are required fields, I think we should allow for a default value on the link text as a minimum here